frontend_lib/src/helper.js

```diff
@@ -516,8 +516,8 @@ export const TIMELINE_TYPE = {
 export const sortTimelineByDate = (timeline) => {
   return timeline.sort((a, b) => {
     // INFO - CH - 20210322 - since we don't have the millisecond from backend, content created at the same second
-    // may very happen. So we sort on content_id in that case. This isn't ideal
-    if (a.created_raw === b.created_raw) return parseInt(a.content_id) - parseInt(b.content_id)
+    // may very happen. So we sort on revision_id in that case. This isn't ideal
+    if (a.created_raw === b.created_raw) return parseInt(a.revision_id) - parseInt(b.revision_id)
```

Comment:
Could you explain this bit?
Why was the timeline sorted by content id and now by revision id?

Reply:
Sorting by content_id does not work for revisions as they all have the same content id. Sorting by revision id does not have the same inconvenient.

Reply:
I completed the comment to explain that.